apps/model-ad/api/config/application-local.yml

```diff
@@ -0,0 +1,2 @@
+openchallenges-challenge-service:
+  welcome-message: 'Welcome to the challenge service (local config).'
```

Comment:
This could be changed to specify model-ad instead of openchallenges. Up to you, since this is primarily for demo purposes!

Reply:
This file is currently not used in this project so I've removed it.